package.json

```diff
@@ -61,9 +58,8 @@
     "open": "0.0.5",
     "promzard": "0.3.0",
     "read": "1.0.7",
-    "request": "2.67.0",
```

Comment:
Hmm, I swear I've done this before (cant' find it) but maybe it was for a project only with ES6 imports.